dbms/src/IO/WriteHelpers.h

```diff
@@ -255,7 +255,7 @@ inline void writeJSONString(const char * begin, const char * end, WriteBuffer &
 }
 
 
-template <char c>
+template <char c, char escape_symbol = '\\'>
```

Comment:
We still have to do something with this name...
because the `escape_symbol` applies only to `c` character and C-style escaping is applied to other characters
and it is counterintuitive.

Reply:
```
template <
    char quote_character, /// Will escape this character in addition to the list of special characters.
    bool escape_quote_with_quote /// Should we escape quote character with quote character ('hello''world') instead of backslash. Other special characters will be escaped with backslash as usual.
    >
```